commands/configure/index.js

```diff
@@ -154,6 +158,18 @@ program
       console.error(e);
       logError('Failed to download files.');
     }
+    try {
+      const { canExportDataTables } = await inquirer.prompt([{ type: 'confirm', name: 'canExportDataTables', message: 'Export data tables now?' }]);
+      if (canExportDataTables) {
+        const exportedTables = await dataTablesExport(null, {}, loadedConfig);
+        if (exportedTables) {
+          logResult('success, Exported all data tables!', 'green');
```

Comment:
some weird capitalization here

Reply:
this literally prints out as `success, Exported all data tables!	green` instead of making the word success green